src/v2/layouts/Button/Button.stories.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { Button, ButtonProps } from './Button';
+import { ComponentMeta, ComponentStory } from '@storybook/react';
+
+export default {
+	title: 'Button',
```

Comment:
Let's move it to design
```suggestion
	title: 'design/Button',
```

Reply:
Resolved